app/src/NativeBase/Components/List.tsx

```diff
@@ -4,13 +4,27 @@
 
 import React, { FC } from 'react'
 import styled from 'styled-components/native'
-import { FlatList } from 'react-native'
 import EventSummary from '@/Components/Event/EventSummary'
-import ProjectSummary from '@/Components/Project/ProjectSummary'
-import { ListType } from '@/Containers/ListContainer'
+import { ListType } from '@/NativeBase/Containers/ListContainer'
 import { navigate, RootStackParamList } from '@/Navigators/utils'
 import { Events, EventsRange } from '@/Services/modules/events'
 import { Projects } from '@/Services/modules/projects'
+import underDevelopmentAlert from '../../Utils/UnderDevelopmentAlert'
+import ColouredTag from '../Components/ColouredTag'
+import MaterialIcons from 'react-native-vector-icons/MaterialIcons'
+
+import {
+  FavouriteIcon,
+  FlatList,
+  Heading,
+  HStack,
+  Pressable,
+  ShareIcon,
+  Text,
+  VStack,
+  Card,
+  Icon,
+} from 'native-base'
 
 // General styled components
 const NoneFound = styled.Text`
```

Comment:
As `NoneFound` and `TryAnotherSearch` are used by projects, please can you also convert these to using NativeBase instead of `styled`.  (No worries about the events-specific things still using `styled`)

Reply:
Actually, don’t worry about changing these — as I’m about to replace how the search results work in SVA-264